Assets/Player/Player.gd

```diff
@@ -0,0 +1,12 @@
+extends Control
+class_name Player
+
+var faction = Global.Factions.NONE
+
+var settlements = []
+var ships = []
+
+func _ready() -> void:
+	print_debug(
+		"I'm {0} in fraction {1} ({2}).".format(
```

Comment:
Spelling error, "fraction". Also, this file could use `#warning-ignore-all:unused_class_variable`